src/react-components/src/components/00-base/Placeholder.tsx

```diff
@@ -0,0 +1,15 @@
+
+import * as React from "react";
+import bem from "../../utils/bem";
+
+export interface PlaceholderProps {
+  modifiers?: string[];
+  blockName?: string;
+}
+
+export default function Placeholder(props: React.PropsWithChildren<PlaceholderProps>) {
```

Comment:
I know this is really "do as I say and not as I do" but it would be really nice to have comments for all the components saying what they do.

Reply:
Updated!